guides/common/modules/ref_additional-resources-project-infrastructure-organization-concepts.adoc

```diff
@@ -0,0 +1,4 @@
+[id="additional-resources-project-infrastructure-organization-concepts_{context}"]
+= Additional resources
+
+* For examples of {Project} deployment, see xref:part-Deployment_Planning[].
```

Comment:
I am not sure if this file add any value to users. While checking this in the rendered docs, I've also realized that the headings is somewhat weird; maybe multiple level 0 headings? I consider this out of scope but I want to have a look at this after your PR.

Reply:
I'd like to see the guide split into two and in such case a link like this would perhaps make more sense?

Reply:
The idea is based on https://github.com/theforeman/foreman-documentation/pull/2866#issuecomment-2233593722. The SOE guide presents the content in a certain order and we can try to emulate that sequence by using next steps and/or additional resources.

The heading level is as I intended it: the additional resources are included on the level of the assembly, so hierarchically they are the same as the included modules.

If this is a non-blocking comment, let's leave it as is right now. But we can of course revisit later to have a proper conversation about it.